docs/configuration/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+id: overview
+sidebar_position: 0
+sidebar_label: Configuration
+description: Learn how to configure Yazi.
+---
+
+# Configuration
+
+There are three configuration files for Yazi:
+
+- [`yazi.toml`](./yazi.md) - Configuration of file management-related functions
+- [`keymap.toml`](./keymap.md) - Configuration of keybindings
+- [`theme.toml`](./theme.md) - Yazi color scheme
+
+Yazi presets these configurations upon release, and uses them by default.
```

Comment:
This sentence doesn't make sense... I prefer what I had written earlier.

Reply:
why! 😭

Reply:
I can't think of a more accurate word than this...

Reply:
Maybe "Yazi 'includes' these configurations upon release, and uses them by default."?

Reply:
I think its much clearer to just say "Yazi's default configuration".